cv_next/app/inactive/page.tsx

```diff
@@ -0,0 +1,9 @@
+import Hero from "@/components/pages/deactivated"
```

Comment:
Please note:
1. "deactivated" is not a semantically-understandable name, especially when imported in an "inactive" page.
2. Don't use the `components/pages` folder; the only thing that's there is temporary. Components that are specific to a page should sit in `[page]/components`.
3. Specifically, I believe the component can be implemented here in the page itself, instead of putting it in a separate file. Unless you'll decide to separate it to smaller components, which'll be great (see notes on that file)